calculate_ims.py

```diff
@@ -243,7 +251,7 @@ def compute_measures_multiprocess(
     :param simple_output:
     :return:
     """
-    converted_comp = convert_str_comp(comp)
+    int_comps, str_comps = convert_str_comp(comp)
```

Comment:
If int comps aren't used in calculate_ims, we should probably move that logic to where it is needed.